ros/api/v1/hosts.py

```diff
@@ -199,7 +199,7 @@ def build_sort_expression(self, order_how, order_method):
         if order_method in score_methods:
             return (
                 sort_order(PerformanceProfile.performance_utilization[
-                    order_method].astext.cast(Integer)),
+                    order_method].astext.cast(Float)),
                 asc(PerformanceProfile.system_id),)
```

Comment:
@PreetiW Change on line 202, will handle the previous sort error, tested :white_check_mark: